.github/workflows/integration-tests.yaml

```diff
@@ -0,0 +1,35 @@
+name: integration-tests
+
+on:
+  # Run integration tests nightly.
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
You might want to use a time that is not a round number since there are often many, many jobs set to run at the exact same time.